rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -50,7 +60,7 @@ public RtmpIOException(int errorCode) {
 
     public void open(String url, boolean isPublishMode) throws RtmpIOException {
         rtmpPointer = nativeAlloc();
-        int result = nativeOpen(url, isPublishMode, rtmpPointer);
+        int result = nativeOpen(url, isPublishMode, rtmpPointer, timeout);
```

Comment:
Ideally these timeout variables will have the unit in the variable name, so it's clear as its passed around what unit it's in. If we can do milliseconds then timeoutMs would be a good name. Ditto elsewhere in this change.

Reply:
Done.